packages/react-dom/src/__tests__/ReactDOMFizzServer-test.js

```diff
@@ -1099,6 +1099,85 @@ describe('ReactDOMFizzServer', () => {
     expect(loggedErrors).toEqual([theError]);
   });
 
+  fit('Errors in boundaries should not be logged when new, unrelated Suspense boundaries commit', async () => {
+    class ErrorBoundary extends React.Component {
+      state = {error: null};
+      static getDerivedStateFromError(error) {
+        return {error};
+      }
+
+      render() {
+        if (this.state.error) {
+          return <Text text="did error" />;
+        }
+        return this.props.children;
+      }
+    }
+
+    const theError = new Error('Boom');
+    function Page() {
+      throw theError;
+    }
+
+    let renderSuspense;
+    function Sibling() {
+      const [state, setState] = React.useState(false);
+      renderSuspense = () => setState(true);
+
+      return state ? <Suspense /> : null;
+    }
+
+    function App({isClient}) {
+      return (
+        <>
+          <ErrorBoundary>
+            <Suspense>
+              <Page />
+            </Suspense>
+          </ErrorBoundary>
+          <Sibling />
```

Comment:
We're triggering an update in the `<Sibling>` tree long after `<ErrorBoundary>` committed yet we still get the "switched to client-side rendering" from the `<ErrorBoundary>` tree.